.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 Thanks for your hard work, please ensure all items are complete before opening a PR.
 
-- [ ] You have signed the CLA as per the instructions in [CONTRIBUTING.md](https://github.com/cloudant/couchbackup/blob/master/CONTRIBUTING.md#contributor-license-agreement)
+- [ ] Tick to sign-off your agreement to the [DCO 1.1](https://github.com/cloudant/couchbackup/blob/master/DCO1.1.txt)
```

Comment:
It's a bit awkward because it's a template. I'm not sure at what level the "draft" PR is when it opens or if it will consistently be at that level. Maybe if there are identifiers in GFM for "current branch" or similar we could do it more easily.